packages/fast-tooling/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/fast-tooling",
-    "description": "FAST Tooling",
+    "description": "A library agnostic set of utilities to assist in creating web UI",
```

Comment:
library agnostic or javascript framework agnostic?

Reply:
or framework-agnostic?  I've heard recently a movement to get away from frameworks altogether, with WC enabling this, it might be good for marketing.

Reply:
You know what, I'm just going to get rid of that entirely, if there is no framework specified I think it's axiomatic.